client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -68,6 +69,19 @@ const CheckoutThankYouHeader = React.createClass( {
 			} );
 		}
 
+		if ( isGuidedTransfer( this.props.primaryPurchase ) ) {
+			if ( typeof this.props.primaryPurchase.meta === 'string' ) {
```

Comment:
Also just noticed that none of the other `if` blocks do a check for `primaryPurchase.meta`. I'm just wondering whats special/different about this case.